shard.yml

```diff
@@ -20,6 +20,6 @@ development_dependencies:
     github: crystal-ameba/ameba
     version: ~> 0.13.4
 
-crystal: 0.35.0
+crystal: 1.0.0
```

Comment:
Maybe this should include `0.35.0` if it's still downward compatible?

Reply:
Good point, actually it's compatible with `0.36.x`so `>= 0.36.0` is fine 👍